src/libs/antares/study/parameters.cpp

```diff
@@ -1074,6 +1074,13 @@ bool Parameters::loadFromINI(const IniFile& ini, uint version, const StudyLoadOp
     if (options.usedByTheSolver)
         prepareForSimulation(options);
 
+    if (options.mpsToExport
+        && (this->include.exportMPS == mpsExportStatus::NO_EXPORT
+            || this->include.exportMPS == mpsExportStatus::UNKNOWN_EXPORT))
+    {
+        this->include.exportMPS = mpsExportStatus::EXPORT_BOTH_OPTIMS;
+    }
```

Comment:
```suggestion
    if (options.mpsToExport)
    {
        this->include.exportMPS = mpsExportStatus::EXPORT_BOTH_OPTIMS;
    }
```

Reply:
I think that we want to force the export of all MPS files if option `--export-mps` is provided by the user.